src/pyOpenMS/pxds/FIAMSScheduler.pxd

```diff
@@ -0,0 +1,20 @@
+from Types cimport *
+from libcpp cimport bool
+from libcpp.map cimport map as libcpp_map
+from libcpp.vector cimport vector as libcpp_vector
+from String cimport *
+
+cdef extern from "<OpenMS/ANALYSIS/ID/FIAMSScheduler.h>" namespace "OpenMS":
+
+    cdef cppclass FIAMSScheduler "OpenMS::FIAMSScheduler":
+        #
+        # wrap-doc:
+        #     ADD PYTHON DOCUMENTATION HERE
+        #
+        FIAMSScheduler() nogil except +
+        FIAMSScheduler(FIAMSScheduler) nogil except +
+        FIAMSScheduler(String filename, String base_dir, bool load_cached_) nogil except +
+        void run() nogil except +
+        libcpp_vector[ libcpp_map[ String, String ] ] getSamples() nogil except +
```

Comment:
this is not going to work in pyOpenMS, we cannot automatically wrap such complex STL constructs. I suggest to comment it out unless you really need it in Python
```suggestion
        # libcpp_vector[ libcpp_map[ String, String ] ] getSamples() nogil except +
```